.github/workflows/release-process.yml

```diff
@@ -0,0 +1,102 @@
+name: kapp-controller release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  kapp-controller-release:
+    name: kapp-controller release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v1
+        with:
+          path: src/github.com/${{ github.repository }}
+
+      - name: Install Carvel Tools
+        uses: vmware-tanzu/carvel-setup-action@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          only: ytt, kbld
+          ytt: v0.37.0
```

Comment:
I'd be tempted to generate these versions from a single source of truth, e.g. write a pre-commit hook that scrapes the versions from our Dockerfile and puts them in here,  as otherwise eventually someone will be confused about some behavior.   granted that's some extra machinery - can be left for the future, no need to block initial rollout of this automation.